docs/SyntaxForModels.mdx

```diff
@@ -32,14 +35,14 @@ p2 = sub, act
 
 And we have the following policy (if in a policy file)
 
-```bash
```

Comment:
Why remove `bash` ? (also the `ini`?)

Reply:
It's not bash
Original doc is https://github.com/casbin/casbin-website/blob/14a4ceba447b8d5ff996319677da79fd2a76c54c/docs/SyntaxForModels.md?plain=1#L35